dev/breeze/src/airflow_breeze/params/shell_params.py

```diff
@@ -267,9 +268,13 @@ def command_passed(self):
     @property
     def mssql_data_volume(self) -> str:
         docker_filesystem = get_filesystem_type("/var/lib/docker")
-        # in case of Providers[....], only leave Providers
-        base_test_type = self.test_type.split("[")[0] if self.test_type else None
-        volume_name = f"tmp-mssql-volume-{base_test_type}" if base_test_type else "tmp-mssql-volume"
+        # Make sure the test type is not too long to be used as a volume name in docker-compose
```

Comment:
We needed that to handle several Providers* tests run in parallel - each parallel docker-compose needs unique volume name